CP02PMCI/CP02PMCI_R00013_ingest.csv

```diff
@@ -0,0 +1,11 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00013/cg_data/stc1/imm/adcpt/adcpt*.DAT,CP02PMCI-RII01-02-ADCPTG010,recovered_host,Expected,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP02PMCI/R00013/instruments/ADCP*/*RDI_000.000,CP02PMCI-RII01-02-ADCPTG010,recovered_inst,Expected,
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00013/cg_data/stc1/irid2shore/stc_status*.txt,CP02PMCI-SBS01-00-STCENG000,recovered_host,Expected,

```

Comment:
There is no irid2shore folder in R00013/cg_data/stc1 in the Raw Data Repository.